www/src/css/custom.css

```diff
@@ -4,7 +4,23 @@
  * work well for content-centric websites.
  */
 
-@import url("https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&family=Space+Mono&display=swap");
```

Comment:
This seems to work fine, but lets google track all traffic to our docs site.

Reply:
I think that's ok